var/spack/repos/builtin/packages/py-tmd/package.py

```diff
@@ -10,10 +10,9 @@ class PyTmd(PythonPackage):
     """Python library for the topological analysis of neurons"""
 
     homepage = "https://github.com/BlueBrain/TMD"
-    url = "https://pypi.io/packages/source/t/tmd/tmd-2.0.3.tar.gz"
+    url = "https://pypi.io/packages/source/t/tmd/tmd-2.0.8.tar.gz"
 
-    version('2.0.4', sha256='b1709f36964d11dd555a35f99dfa26187ce2ef98a56f07eaa4805da944e86652', preferred=True)
-    version('2.0.3', sha256='ffde39a2fa6221a093ff66390a205149348e97a459aba1ab8f04998070e00dc1')
+    version('2.0.8', sha256='b1709f36964d11dd555a35f99dfa26187ce2ef98a56f07eaa4805da944e86652')
```

Comment:
Everything else is tagged :)

Now why is the SHA the same, but the version bumped?

Reply:
This one comes from PyPi, hence the tags.
I have updated the new SHA.
And I have changed my mind about keeping `@space2` in fact :)

Reply:
Ah, sorry, didn't realize that. Thanks for fixing the sha!